src/c++/perf_analyzer/genai-perf/genai_perf/llm_metrics.py

```diff
@@ -36,7 +36,7 @@
 import numpy as np
 import pandas as pd
 from genai_perf.tokenizer import Tokenizer
-from genai_perf.utils import load_json, remove_sse_prefix
+from genai_perf.utils import load_json, remove_sse_prefix, scale
```

Comment:
I think scale needs to be renamed and have a comment explaining what it does attached. I know this is out of bounds for this PR, but scale is unclear for me as a name. If you could do a find and replace here, that would be helpful.

Reply:
Honestly, looking at it, it seems necessary for me. It's a more complex way of doing a multiplication that offers nothing extra.
`6 * 100` is clearer than `scale(6, 100)` IMO.